tests/circuit_drawer/test_circuit_drawer.py

```diff
@@ -297,20 +300,20 @@ def qfunc(a, b, c, angles):
     dev = qml.device("default.qubit", wires=5)
 
     qnode = qml.QNode(qfunc, dev)
-    qnode._construct((0.1, 0.2, 0.3, np.array([0.4, 0.5, 0.6])), {})
-    qnode.evaluate((0.1, 0.2, 0.3, np.array([0.4, 0.5, 0.6])), {})
```

Comment:
`_construct` and `evaluate` do not exist in tape mode. Replaced by simply calling the QNode, which performs the same logic.